### PR TITLE
More complex conditional example

### PR DESCRIPTION
**Pull request by keithmantell**
As a bit of an elisp newbie I liked this more complex example!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,6 +247,15 @@ graphical Emacs, not for other Emacsen I may start at the command line:
   (add-hook 'after-init-hook 'server-start t)
   (add-hook 'after-init-hook 'edit-server-start t))
 ```
+In another example, we can load things conditional on the operating system:
+
+```
+(use-package exec-path-from-shell
+  :if (string-equal system-type "darwin")
```
Comment (npostavs) [inline README.md]: Use `(eq system-type 'darwin)` though.

Comment (keithmantell) [inline README.md]: Great! I did say i was a newbie!

New PR required?

Comment (npostavs) [inline README.md]: This is a very minor change, so IMO it should be okay to amend the commit and then `push --force` to update this PR.

**Comment by jwiegley** [comment]
Thanks!

**Comment by nickmccurdy** [comment]
Please see #452, I updated the predicate to the recommended one that detects if Emacs is running in a terminal (in which case you don't need to load `exec-path-from-shell`).

**Pull request merged**